src/test/groovy/myTest.groovy

```diff
@@ -0,0 +1,11 @@
+import spock.lang.Specification
+
+/**
+ * Created by mowall on 14/9/16.
+ */
+class myTest extends Specification {
```

Comment:
As git will not take empty dir
and you said to maintain the structure for working directory.
-src
   -main
   -test
So i had add sample class in test folder